src/test/java/qna/domain/AnswerTest.java

```diff
@@ -1,6 +1,70 @@
 package qna.domain;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.util.Optional;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
 public class AnswerTest {
-    public static final Answer A1 = new Answer(UserTest.JAVAJIGI, QuestionTest.Q1, "Answers Contents1");
-    public static final Answer A2 = new Answer(UserTest.SANJIGI, QuestionTest.Q1, "Answers Contents2");
+
+  @Autowired
+  AnswerRepository answers;
+
+  public static final Answer A1 = new Answer(UserTest.JAVAJIGI, QuestionTest.Q1,
+      "Answers Contents1");
+  public static final Answer A2 = new Answer(UserTest.SANJIGI, QuestionTest.Q1,
+      "Answers Contents2");
```

Comment:
```suggestion
  public static final Answer A1 = new Answer(UserTest.JAVAJIGI, QuestionTest.Q1,
      "Answers Contents1");
  public static final Answer A2 = new Answer(UserTest.SANJIGI, QuestionTest.Q1,
      "Answers Contents2");
      
  @Autowired
  AnswerRepository answers;
```

클래스 변수 이후 인스턴스 변수가 오는 것이 관례입니다!